packages/realm/package.json

```diff
@@ -31,7 +31,7 @@
     "url": "https://www.mongodb.com/docs/realm/"
   },
   "config": {
-    "anonymizedBundleId": "tkgif/+3l1e9wStGJp2TOngAK3UcQ2u7OM8ZYJU5JYo="
+    "anonymizedBundleId": "1RmJBlqbKuzyRiPm4AsdIIxe8xlRUntGcGFEwUnUh6A="
```

Comment:
I added a comment about this elsewhere, but can't find it. We should do a follow up PR to skip updating this field of `REALM_DISABLE_ANALYTICS` is set.

Reply:
I included that in #6313 👍 I'll get this change reverted ...

Reply:
```suggestion
    "anonymizedBundleId": "tkgif/+3l1e9wStGJp2TOngAK3UcQ2u7OM8ZYJU5JYo="
```